featuretools/tests/synthesis/test_dask_dfs.py

```diff
@@ -0,0 +1,369 @@
+
+    dask_fm, _ = ft.dfs(entityset=dask_es,
+                        target_entity="data",
+                        trans_primitives=primitives_list,
+                        cutoff_time=cutoff_times)
+
+    pd_es = ft.EntitySet(id="pd_es")
+    pd_es.entity_from_dataframe(entity_id="data",
+                                dataframe=df,
+                                index="id",
+                                time_index="dates",
+                                variable_types={"strings": ft.variable_types.NaturalLanguage})
+
+    fm, _ = ft.dfs(entityset=pd_es,
+                   target_entity="data",
+                   trans_primitives=primitives_list,
+                   cutoff_time=cutoff_times)
+    # Because row ordering with Dask is not guaranteed, we need to sort on two columns to make sure that values
+    # for instance id 0 are compared correctly. Also, make sure the boolean column has the same dtype.
+    fm = fm.sort_values(['id', 'labels'])
+    dask_fm = dask_fm.compute().set_index('id').sort_values(['id', 'labels'])
+    dask_fm['IS_WEEKEND(dates)'] = dask_fm['IS_WEEKEND(dates)'].astype(fm['IS_WEEKEND(dates)'].dtype)
+    pd.testing.assert_frame_equal(fm, dask_fm)
+
+
+def test_single_table_dask_entityset_dates_not_sorted():
+    dask_es = EntitySet(id="dask_es")
+    df = pd.DataFrame({"id": [0, 1, 2, 3],
+                       "values": [1, 12, -34, 27],
+                       "dates": [pd.to_datetime('2019-01-10'),
+                                 pd.to_datetime('2019-02-03'),
+                                 pd.to_datetime('2019-01-01'),
+                                 pd.to_datetime('2017-08-25')]})
+
+    primitives_list = ['absolute', 'is_weekend', 'year', 'day']
+    values_dd = dd.from_pandas(df, npartitions=1)
+    vtypes = {
+        "id": ft.variable_types.Id,
+        "values": ft.variable_types.Numeric,
+        "dates": ft.variable_types.Datetime,
+    }
+    dask_es.entity_from_dataframe(entity_id="data",
+                                  dataframe=values_dd,
+                                  index="id",
+                                  time_index="dates",
+                                  variable_types=vtypes)
+
+    dask_fm, _ = ft.dfs(entityset=dask_es,
+                        target_entity="data",
+                        trans_primitives=primitives_list,
+                        max_depth=1)
+
+    pd_es = ft.EntitySet(id="pd_es")
+    pd_es.entity_from_dataframe(entity_id="data",
+                                dataframe=df,
+                                index="id",
+                                time_index="dates")
+
+    fm, _ = ft.dfs(entityset=pd_es,
+                   target_entity="data",
+                   trans_primitives=primitives_list,
+                   max_depth=1)
+
+    pd.testing.assert_frame_equal(fm, dask_fm.compute().set_index('id').loc[fm.index])
+
+
+def test_secondary_time_index():
```

Comment:
let's change this to `test_dask_entityset_secondary_time_index` to be consistent

Or we could drop the `dask_entityset` from all of the test case names and rely on the  test name + file name to infer, but I think that approach would be harder to quickly parse from pytest results

Reply:
renamed